### PR TITLE
Format Help Command Output

### PR DESCRIPTION
**Pull request by BlackFrog1**
Word-wrap the output of each command description via the help command. The Help Command Test Unit was updated for the new layout.
 
Window Width @ 80 character - standard
Window Wisth @ 40 character - extreme example

### CODE REVIEW

File: src/ScriptCs.Core/App_Packages/LibLog.3.1/LibLog.cs [hunk 1]
```diff
@@ -1814,7 +1814,7 @@ internal static class LogMessageFormatter
         /// <summary>
         /// Some logging frameworks support structured logging, such as serilog. This will allow you to add names to structured data in a format string:
         /// For example: Log("Log message to {user}", user). This only works with serilog, but as the user of LibLog, you don't know if serilog is actually 
-        /// used. So, this class simulates that. it will replace any text in {curlybraces} with an index number. 
+        /// used. So, this class simulates that. it will replace any text in {curly braces} with an index number. 
         /// 
         /// "Log {message} to {user}" would turn into => "Log {0} to {1}". Then the format parameters are handled using regular .net string.Format.
         /// </summary>
```
Comment (adamralph) [inline src/ScriptCs.Core/App_Packages/LibLog.3.1/LibLog.cs]: This file is imported by the LibLog package so we really shouldn't edit it. Any changes here should be submitted as a PR to https://github.com/damianh/LibLog

**Comment by adamralph** [comment]
@BlackFrog1 this is a nice change, thanks!

@scriptcs/core it should be noted that it is a breaking API change, i.e. the addition of `IConsole.Width` so we'll need an opinion on whether we want to take this.

**Comment by filipw** [comment]
take it it's a small breaking change

**Comment by adamralph** [comment]
Yep, it will only break third party implementations of `IConsole`.

**Comment by khellang** [comment]
If we only had https://github.com/dotnet/roslyn/issues/258 :wink:

**Comment by adamralph** [comment]
This PR is waiting on two things:

1. @BlackFrog1 to revert the changes to LibLog.cs
1. @scriptcs/core to decide if we are OK with the breaking change to `IConsole`

**Comment by khellang** [comment]
Personally I don't really care about breaking stuff. I don't think many people will get "broken" by adding a member to an interface.

**Comment by glennblock** [comment]
Adding a member to an interface breaks all implementers. In the case of
IConsole this is likely custom hosters which may not be so bad. That is not
always the case though. Adding a member to IScriptPack for example will
have much more serious consequences.

**Comment by BlackFrog1** [comment]
What is the best course of action in reverting the change to LibLog.cs? Do I take all the changes back?  Or do I just resubmit the change for just this file?

**Comment by adamralph** [comment]
@BlackFrog1 yes, that looks fine now, thanks.

**Comment by BlackFrog1** [comment]
I think I finally got the hang of creating PR.  Once I create cleaner PR for the items in #1032, I will delete/close this.  Give me a few days.

**Comment by adamralph** [comment]
Thanks @BlackFrog1, we'll await your new PR.

**Pull request closed without merging**